source/helpers/copy-bulk-files.js

```diff
@@ -1,7 +1,10 @@
 /* eslint-disable global-require, no-console */
 
+import Debug from 'debug';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['debug' should be listed in the project's dependencies. Run 'npm i -S debug' to add it](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5031740)